kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/TypedValues.java

```diff
@@ -19,8 +19,10 @@
 /**
  * The Class TypedValues is an utility class to quickly create different
  * {@link TypedValue}
+ * 
+ * @noextend This class is not intended to be extended by clients.
  */
-public final class TypedValues {
+public class TypedValues {
```

Comment:
As this is an utility class, I strongly prefer to have final as consumers can extend this by adding new methods to it without breaking any functionalities. As there won't be any instances of this class, it will not be extended by other classes. Such extensions for utility classes are not at all common in Java community.